keps/sig-autoscaling/5325-hpa-pod-selection-accuracy/kep.yaml

```
@@ -0,0 +1,47 @@
+title: KEP Template
+kep-number: 5325
+authors:
+  - "@omerap12"
+  - "@adrianmoisey"
+owning-sig: sig-autoscaling
+participating-sigs: []
+status: provisional #|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-05-21
+reviewers:
+  - TBD
+  - "@alice.doe"
+approvers:
+  - TBD
+  - "@oscar.doe"
```

I'm never sure how to fill these in, but this makes sense to me:

```suggestion
  - "@soltysh"
approvers:
  - "@soltysh"
```